FiftyOne.Pipeline.Core/FlowElements/PipelineBuilder.cs

```diff
@@ -222,7 +230,7 @@ private void GetAvailableElementBuilders()
         /// <see cref="PipelineOptions"/> instance.
         /// </param>
         private void AddElementToList(
```

Comment:
This is cosmetics, but I would rename this method into `EnqueueElement` now since it is no longer adding to the list, but adding to the queue.